apps/system/js/service.js

```diff
@@ -35,6 +35,10 @@
     /**
      * Request a service and get a promise.
      * The service name may include the name of server or not if it is unique.
+     *
+     * The request and query format are different,
+     * request does not accept getter format.
+     *
```

Comment:
I've struggled with this fact for a while, so it worth to add some comment here for reference.